run_client.py

```diff
@@ -2,7 +2,6 @@
 import os
 import sys
 import logging
-
```

Comment:
This newline was there on purpose. It separates builtin imports from seekers imports.